projects/go-tester/src/app/components/off-canvas-test/off-canvas-test.component.html

```diff
@@ -1,7 +1,4 @@
 <div class="go-container">
-  <go-off-canvas-header-actions>
-    <go-button>Close</go-button>
-  </go-off-canvas-header-actions>
```

Comment:
This is breaking the build, it is fixed with a rebase to `dev`.. but I didn't want to do that in this PR.